docker/data-science/notebooks/ch07/1.index-datasets.ipynb

```diff
@@ -321,7 +321,8 @@
    ],
    "source": [
     "entitites_collection = engine.create_collection(\"entities\")\n",
```

Comment:
pre-existing issue, but we should fix variable name from "entitiTes" to "entities".